app/proxy/const.go

```diff
@@ -140,12 +140,3 @@ var ignoreLog = []string{
 	MethodAccountBalance,
 	MethodStatus,
 }
-
-func shouldLog(method string) bool {
```

Comment:
Seems like I forgot to add an actual call for this, I would reinstate it, logs are swamped with `wallet_balance`.

Reply:
esp because that's our only way to see what's going on in the cluster right now